packages/blocks/src/block-content-provider/index.js

```diff
@@ -1,20 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { Component, RawHTML } from '@wordpress/element';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { createContext, RawHTML } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { serialize } from '../api';
 
+const { Consumer, Provider } = createContext( {
+	BlockContent: () => {},
+} );
+
 /**
  * An internal block component used in block content serialization to inject
  * nested block content within the `save` implementation of the ancestor
  * component in which it is nested. The component provides a pre-bound
  * `BlockContent` component via context, which is used by the developer-facing
  * `InnerBlocks.Content` component to render block content.
  *
+ * @return {WPElement} Element with BlockContent injected via context.
```

Comment:
Minor: We might want to establish conventions around tag order. How I've treated it, I always have `@return` last, preceded by `@param`, preceded by anything else.

Reply:
Yes, makes sense. In general, we could make JSDoc validation more strict since we started using it more consistently.